pipelines-it-remote.yml

```diff
@@ -0,0 +1,64 @@
+jobs:
+- job: 'Ubuntu_16_04_remote'
```

Comment:
Please use a meaningful name to describe the job

Reply:
changed to 'integration_test_remote'